charts/kube-prometheus-stack/templates/alertmanager/ingress.yaml

```diff
@@ -4,13 +4,8 @@
 {{- $servicePort := .Values.alertmanager.service.port -}}
 {{- $routePrefix := list .Values.alertmanager.alertmanagerSpec.routePrefix }}
 {{- $paths := .Values.alertmanager.ingress.paths | default $routePrefix -}}
-{{- if .Capabilities.APIVersions.Has "networking.k8s.io/v1" -}}
-apiVersion: networking.k8s.io/v1
-  {{- else if .Capabilities.APIVersions.Has "networking.k8s.io/v1beta1" -}}
-apiVersion: networking.k8s.io/v1beta1
-  {{- else -}}
-apiVersion: extensions/v1beta1
-  {{- end }}
+{{- $apiIsStable := include "kube-prometheus-stack.ingress.isStable" . -}}
```

Comment:
This does not work as intended.

`$apiIsStable` will evaluate to a string with value `"true"` or `"false"`, which then both evaluate to a boolean TRUE value on lines 41 and 61.

Reply:
Change it to:
```
{{- $apiIsStable := eq (include "kube-prometheus-stack.ingress.isStable" .) "true" -}}
```

Reply:
That is unfortunate that it arbitrarily quotes with include while simultaneously stripping quotes with toYaml. Untested but id bet because of that inconsistency you could alao just do | toYaml and not have to eq check